# This file is part of HiGFXback

# requires
REQUIRES="autotools-wrappers-am-build ncurses-build"

pkg-config --exists --print-errors $REQUIRES || exit 1

# configure (ac-2.60; am-1.9)
aclocal -I m4; autoheader; autoconf; automake -a -c
./configure --prefix=

# build
make

# install
install -d $DESTDIR/bin
install src/nano $DESTDIR/bin

# build.pc
install -d $DESTDIR/share/pkgconfig
cat > $DESTDIR/share/pkgconfig/nano-build.pc << EOF
Name: nano
Version: 2.0.6
Description: Pico editor clone with enhancements
Requires: $REQUIRES

exec=\\
/bin/nano
EOF

exit

# patch
--- nano-2.0.6.orig/configure.ac
+++ nano-2.0.6/configure.ac
@@ -23,6 +23,7 @@
 AC_CONFIG_SRCDIR([src/nano.c])
 AC_CANONICAL_TARGET([])
 AM_INIT_AUTOMAKE
+AM_MAINTAINER_MODE
 AC_CONFIG_HEADERS([config.h])
 
 AC_PREREQ(2.54)

# source
https://nano-editor.org/dist/v2.0/nano-2.0.6.tar.gz
